lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +27,40 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
```

Comment:
I was getting more at the inconvenience of having to call `getattr` a bunch of times instead of being able to access a property directly, but it's very minor. I'm happy to keep things smaller.